SoulMatchServer-main/src/main/java/com/soulmatch/authentication/LoginController.java

```diff
@@ -25,8 +27,13 @@ public class LoginController {
     public ResponseEntity<?> loginUser(@RequestBody User user) {
         User databaseUser = controller.getUserByEmail(user.getEmail());
```

Comment:
shoudn't this be User databaseUser = databaseUser.getUserByEmail(user.getEmail()); ?